.eslintrc

```diff
@@ -7,9 +7,16 @@
   },
   "rules": {
     // Allowing cyclic dependancies
-    "import/no-cycle": "off"
+    "import/no-cycle": "off",
```

Comment:
Just as a comment for others here. I also run into an issue here for `eslint`. I am not sure root-cause but, it seems like the structure of the npm package. We should try to fix this if we can but, I don't think we should spend the time to do that right now